lib/slayer/form.rb

```diff
@@ -1,17 +1,15 @@
+require 'virtus'
```

Comment:
Shhh... hound. Shhh.

I've added disabling this check to the rubocop file in my rubocop cleanup pull request.